README.md

```diff
@@ -1,2 +1,19 @@
-# aws-consumer-service
-Consumer Service for Screwdriver AWS Integration
+# Screwdriver AWS Consumer Service
+
+> Consumer-Service Package for Screwdriver AWS Integration
+
+This is the package for Screwdriver continuous delivery solution that receives and processes build messages.
+
+## Usage
+
+```bash
+go get aws-consumer-service
```

Comment:
```suggestion
go install github.com/screwdriver-cd/aws-consumer-service@latest
```

`go get` is deprecated for latest go version so we should use `go install`.
https://golang.org/doc/go-get-install-deprecation